### PR TITLE
[DSCP-193] Fixed accidentally broken config

### PR DESCRIPTION
**Pull request by Heimdell**
### Description

Fixing config I broke when doing DSCP-193

### YT issue

https://issues.serokell.io/issue/DSCP-354

### Checklist

Possible related changes (conditional):
- [ ] I added tests if required, in case they are:
  - Covering new introduced functionality (feature).
  - Regression tests preventing the bug from silently reappearing again (bugfix).
- [x] I have checked the [sample config](../tree/master/docs/config-full-sample.yaml) and [launch scripts](../tree/master/scripts/launch) and updated them if it was necessary (especially if executables or CLIs were changed).
- [x] I have checked READMEs and changed them accordingly if it's necessary (the main [README.md](../tree/master/README.md) as well as subproject READMEs for all related executables).
- [ ] If a public API structure or/and data serialization was changed, I made sure that these changes are reflected in the:
  - [Swagger](../tree/master/specs/disciplina) API specs.
  - Any [related documentation](../tree/master/docs/api-types.md).
- [ ] If any public documentation was written, I have spellchecked it.

Stylistic (obligatory):
- [x] My commit history is clean, descriptive and do not contain merge or revert commits or commits like "WIP".
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).
- [x] My code is documented (haddock) and these docs are decent (full enough and spellchecked).

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -36,11 +36,6 @@ demo: &demo
           - # Test key #2, secret: fOhZOFr6SSnYv1/xfKPPMHdWLdQDboysJ4Q794Gezmw=
             - 3BAyX5pSiFerJgLsJZ9XyoBDMHPtq3NLqm5AGPcBpba6W644zSiSWy3vVr
             - 1000
-  witness:
-    db:
-      path: "/var/db/dscp-witness"
-      clean: false
-
     logging:
```
Comment (pasqu4le) [inline config.yaml]: I don't understand this change. This `logging` here should be a child of `witness` (see the educator component below), but with this it becomes a child of `core`.

File: config.yaml [hunk 2]
```diff
@@ -70,6 +65,11 @@ demo: &demo
               log:
                 severity: Debug+
 
+  witness:
+    db:
+      path: "/var/db/dscp-witness"
```
Comment (mkaito) [inline config.yaml]: This needs to be overridden for production, so not much point in putting it here.

File: docs/config-full-sample.yaml [hunk 3]
```diff
@@ -97,6 +97,7 @@ sample:
     db:
       # Path to the database folder
       path: "/var/db/dscp-witness"
+      clean: False
```
Comment (mkaito) [inline docs/config-full-sample.yaml]: So what does this do?

**Comment by flyingleafe** [comment]
@mkaito @Heimdell
It seems to me that the problem has been resolved by other means for a while and therefore there's no point in keeping this PR.

**Pull request closed without merging**